src/chart.zoom.js

```diff
@@ -164,11 +164,23 @@ function doZoom(chartInstance, zoom, center) {
 		var zoomMode = helpers.getValueOrDefault(chartInstance.options.zoom.mode, defaultOptions.zoom.mode);
 		zoomOptions.sensitivity = helpers.getValueOrDefault(chartInstance.options.zoom.sensitivity, defaultOptions.zoom.sensitivity);
 
+		// Which axe should be modified when figers were used.
+		if (zoomMode == 'xy' && whichAxes !== undefined)
+		{
```

Comment:
We use the style below. Can you update the braces?

```
if (condition) {

} else {

}
```

Reply:
Should be fixed now.